colcon_in_container/verb/release_in_container.py

```diff
@@ -202,14 +231,19 @@ def main(self, *, context):  # noqa: D102
                                       f'failed: {str(e)}')
                 logger.info(f'Package {package} released!')
 
+                if context.args.install_debs:
```

Comment:
should this be `if context.args.install_released_packages`?

Reply:
it was changed https://github.com/canonical/colcon-in-container/pull/40/commits/b36f5ffea7e8d1a60881adb3f41b77cab3f524cd and this was missed probably